src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Dfc.ProviderPortal.Apprenticeships.Models
+{
+    public class SPResponse
```

Comment:
Is this class used? If not it can go away

Reply:
Yes it being used to return spresponse
 var response = await client.ExecuteStoredProcedureAsync<SPResponse>(UriFactory.CreateStoredProcedureUri(_settings.DatabaseId, collectionId, "UpdateRecordStatuses"), requestOptions, UKPRN, currentStatus, statusToBeChangedTo);

Reply:
Doesn't appear to actually be used there...

Reply:
removed